src/core/Akka.Streams/Stage/GraphStage.cs

```diff
@@ -423,6 +425,16 @@ public Timer(int id, ICancelable task)
     /// </summary>
     public abstract class GraphStageLogic : IStageLogging
     {
+        private readonly Queue<ActorGraphInterpreter.AsyncInput> _callbacksWaitingForInterpreter = new();
```

Comment:
Queue to hold early callbacks that were invoked before the interpreter is initialized